modules/controlfreec/1.1/config/default.yaml

```diff
@@ -17,41 +17,53 @@ lcr-modules:
         options:
             configFile: "{MODSDIR}/config/freec/config_WGS.txt"
             BedGraphOutput: FALSE
-            breakPointThreshold: 0.8 #use 0.6 for more segments to get more CNVs
+            breakPointThreshold: 0.6 #use 0.6 for more segments to get more CNVs
             breakPointType: 2
-            coefficientOfVariation: 0.05
+            coefficientOfVariation: 0.062
             contamination: 0 # a priori known value of tumor sample contamination by normals
-            contaminationAdjustment: FALSE
-            degree: "3&4" #3&4 for WGS; 1 for WES
-            forceGCcontentNormalization: 0 #0 for WGS; 1 for WES
-            minCNAlength: 1 #1 for WGS; 3 for WES
+            contaminationAdjustment: TRUE
+            degree: '3\&4' #3&4 for WGS; 1 for WES
+            forceGCcontentNormalization: 1 #0 for WGS; 1 for WES # 0 forces control-base normalization, 1 forces GC
+            intercept: 0 # 0 for control-based (paired) ; 1 for GC-content (unpaired)
+            minCNAlength: 8 #1 for WGS; 3 for WES
+            minimalCoveragePerPosition: 10 # default is 0
+            minMappabilityPerWindow: 0.9
+            minQualityPerPosition: 20
             minimalSubclonePresence: 20 #20 for WGS; 30 for WES
             maxThreads: 24
-            noisyData: FALSE #set TRUE for exomes to avoid false positives due to non-uniform capture
-            ploidy: 2,3,4 #will select the ploidy that explains the most CNAs
+            noisyData: TRUE #set TRUE for exomes to avoid false positives due to non-uniform capture
+            ploidy: 2 #will select the ploidy that explains the most CNAs
+            shiftInQuality: 0
             #sex: XX or XY
-            step: 10000 # only if window is specified; do not use for WES
+            step: 20000 # only if window is specified; do not use for WES
             telocentromeric: 50000
-            window: 50000
+            uniqueMatch: TRUE
+            window: 100000
+
 
         software:
             FREEC_sig: "{MODSDIR}/etc/scripts/assess_significance.R"
             FREEC_graph: "{MODSDIR}/etc/scripts/makeGraph.R"
             FREEC_graph_chr: "{MODSDIR}/etc/scripts/makeGraph_Chromosome.R"
+            freec2bed: "{MODSDIR}/etc/scripts/freec2bed.pl"
 
         
         threads:
             controlfreec_run: 24
             calc_sig: 1
             plot: 1
+            freec2bed: 1
 
         mem_mb:
-            controlfreec_run: 8000
+            pileup: 48000
+            cat: 8000
+            controlfreec_run: 24000
             calc_sig: 1000
             plot: 1000
+            freec2bed: 1000
         
         pairing_config:
             genome:
```

Comment:
in theory this tool works for WES, so exomes should be configured here. In current pairing config I think it will not run on exomes. You can test it on the WES sample from demo, even in dry run.

Reply:
Yes, it should work on WES, but it has to be in paired mode. I have not tried testing it extensively yet though. The current configuration requires a separate config_WGS.txt file (to include a "target_regions" section for the exons/target sites).